### ISSUE TITLE
weird treatment of dates on papers that needed moderation

### ISSUE THREAD

**Issue by bentoner** [scirate/scirate#170]
If you compare http://arxiv.org/list/physics.gen-ph/recent and https://scirate.com/arxiv/physics.gen-ph?range=20 they don't match. Look at a paper like http://arxiv.org/abs/1402.7033 which was submitted on Tue, 4 Feb 2014 11:33:45 GMT but didn't appear until the Fri, 28 Feb 2014 mailing, presumably because it was waiting on someone at the arXiv to approve it.

We currently just throw the paper in with the old papers from the submission date, so it wouldn't be seen by someone following an arxiv category via scirate. Does this matter? Do delays like this happen for less speculative papers too?

@mispy i made an issue to record this but i would close it wontfix unless there's evidence that it's a real problem
